bigbluebutton-html5/imports/startup/client/intl.jsx

```diff
@@ -157,14 +162,15 @@ class IntlStartup extends Component {
   render() {
     const { fetching, normalizedLocale, messages } = this.state;
     const { children } = this.props;
+    const { formattedLocale } = formatLocaleCode(normalizedLocale);
 
     return (
       <>
         {(fetching || !normalizedLocale) && <LoadingScreen />}
 
         {normalizedLocale
           && (
-          <IntlProvider fallbackOnEmptyString={FALLBACK_ON_EMPTY_STRING} locale={normalizedLocale} messages={messages}>
+          <IntlProvider fallbackOnEmptyString={FALLBACK_ON_EMPTY_STRING} locale={formattedLocale} messages={messages}>
```

Comment:
```suggestion
          <IntlProvider fallbackOnEmptyString={FALLBACK_ON_EMPTY_STRING} locale={normalizedLocale} messages={messages}>
```

See the comment above.